module/module.go

```diff
@@ -554,6 +665,100 @@ func (m *Module) OperationManager() *operation.Manager {
 	return m.operations
 }
 
+func (m *Module) ListStreams(ctx context.Context, req *streampb.ListStreamsRequest) (*streampb.ListStreamsResponse, error) {
+	// names := make([]string, 0, len(m.streamSourceByName))
+	// for _, n := range maps.Keys(m.streamSourceByName) {
+	// 	names = append(names, n.String())
+	// }
+	// return &streampb.ListStreamsResponse{Names: names}, nil
+	return &streampb.ListStreamsResponse{}, nil
+}
+
+func (m *Module) AddStream(ctx context.Context, req *streampb.AddStreamRequest) (*streampb.AddStreamResponse, error) {
+	name, err := resource.NewFromString(req.GetName())
+	if err != nil {
+		return nil, err
+	}
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	if m.pc == nil {
+		return nil, errors.New("module has no peer connection")
+	}
+	// vcss, ok := m.streamSourceByName[name]
+	// if !ok {
+	// 	return nil, errors.Errorf("unknown stream for resource %s, this is what I got: %#v", name.String(), m.streamSourceByName)
+	// }
+
+	// if _, ok = m.activeResourceStreams[name]; ok {
+	// 	m.logger.Warn("(m *Module) AddStream called with %s when there is already a stream for peer connection %p", req.GetName(), m.pc)
+	// 	return &streampb.AddStreamResponse{}, nil
+	// }
+
+	// sub, err := camera.NewVideoCodecStreamSubscription(512)
+	// if err != nil {
+	// 	return nil, errors.Wrap(err, "error creating stream")
+	// }
+	tlsRTP, err := webrtc.NewTrackLocalStaticRTP(webrtc.RTPCodecCapability{MimeType: "video/H264"}, "video", name.String())
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating a new TrackLocalStaticRTP")
+	}
+
+	// TODO call remove track on error
+	sender, err := m.pc.AddTrack(tlsRTP)
+	if err != nil {
+		return nil, errors.Wrap(err, "error adding track")
+	}
+	// err = vcss.SubscribeRTP(ctx, sub, func(pkts []*rtp.Packet) error {
+	// 	for _, pkt := range pkts {
+	// 		if err := tlsRTP.WriteRTP(pkt); err != nil {
+	// 			m.logger.Warn(err.Error())
+	// 		}
+	// 	}
+	// 	return nil
+	// })
+
+	if err != nil {
```

Comment:
I suppose this doesn't matter, but I assume this was intended to be commented out?